doc/source/skips/0-governance.rst

```diff
@@ -0,0 +1,151 @@
+which will conclude one month from the call for the vote. Any vote must be
+backed by a :ref:`scikit-image proposal (SKIP) <skip>`. If no option can gather
+two thirds of the votes cast, the decision is escalated to the TC, which in
+turn will use consensus seeking with the fallback option of a simple majority
+vote if no consensus can be found within a month. This is what we hereafter may
+refer to as “the decision making process”.
+
+Decisions (in addition to adding core developers and TC membership as above)
+are made according to the following rules:
+
+- **Minor documentation changes**, such as typo fixes, or addition / correction of a
+  sentence (but no change of the scikit-image.org landing page or the “about”
+  page), require approval by a core developer *and* no disagreement or requested
+  changes by a core developer on the issue or pull request page (lazy
+  consensus). Core developers are expected to give “reasonable time” to others
+  to give their opinion on the pull request if they’re not confident others
+  would agree.
+
+- **Code changes and major documentation changes** require agreement by *two*
+  core developers *and* no disagreement or requested changes by a core developer
+  on the issue or pull-request page (lazy consensus).
+
+- **Changes to the API principles and changes to dependencies or supported
+  versions** require a :ref:`skip` and follow the decision-making process
+  outlined above.
+
+- **Changes to this governance model or our mission, vision, and values**
+  require a :ref:`skip` and follow the decision-making process outlined above.
+
+If a veto -1 vote is cast on a lazy consensus, the proposer can appeal to the
+community and core developers and the change can be approved or rejected using
+the decision making procedure outlined above.
+
+.. _skip:
+
+Improvement proposals (SKIPs)
+=============================
+For all votes, a formal proposal must have been made public and discussed before the
+vote. The lifetime of a SKIP is as follows:
+
+- A proposal is brought up as either a GitHub issue or a post to the mailing
+  list.
+- After sufficient discussion, the core advocate(s) of the proposal must consolidate
+  the discussion into a single document, with appropriate references to the
+  original discussion, representing the pros and cons brought up by each
+  participant.
+- Such proposal must be a consolidated document, in the form of a
+  ‘SciKit-Image Proposal’ (SKIP), rather than a long discussion on a GitHub issue or
+  the mailing list.
+- To submit a SKIP, you should copy the `SKIP template
+  <https://github.com/scikit-image/scikit-image/tree/master/doc/source/skips/template.rst>`_,
+  and give it a new name in the same directory, for example,
+  ``35-currying-all-functions.rst``. You should then fill in each section with
+  appropriate links to prior discussions. Finally, you should submit the added
+  file as a pull request (see the :ref:`contributing guide <howto_contribute>`).
```

Comment:
what happens when the changes suggested in the proposal have been implemented?

Reply:
The PR that implements them, whether it is the same PR or a subsequent one, updates the SKIP status. I'll add a line for this.

Reply:
done